src/components/AssetsComponent.jsx

```diff
@@ -55,7 +63,9 @@ export class AssetsComponent extends Component {
               <Table.HeaderCell>
                 <div className="header">
                   Category
-                  <ModalComponent />
+                  <ModalComponent modalTitle="Add Asset Category">
+                    {this.addcategory()}
```

Comment:
Why don't you just use the component directly here? `addcategory` and `addAssetmodel` methods are unnecessary

Reply:
done